### PR TITLE
[GSoC] Update Dialog and Fix failing tests

### PR DESCRIPTION
**Pull request by 4D17Y4**
Update Custom Selector Welcome Dialog to include the preview feature.
Fix Failing tests.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -646,7 +646,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="back">Back</string>
   <string name="welcome_custom_picture_selector_text">Welcome to Custom Picture Selector</string>
   <string name="custom_selector_info_text1">This picker shows differently pictures that are already to Commons.</string>
-  <string name="custom_selector_info_text2">Unlike the picture on the left, the picture on the right has the Commons logo indicating it is already uploaded.</string>
+  <string name="custom_selector_info_text2">Unlike the picture on the left, the picture on the right has the Commons logo indicating it is already uploaded. \n Hold tap for image preview.</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:649]: I think `Touch & hold` is the most common term? https://support.google.com/accessibility/android/answer/6006989?hl=en

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (4D17Y4) [reply app/src/main/res/values/strings.xml]: Done.

**Review by madhurgupta10 — APPROVED** [review]
This fixes the FAB failing test, thanks!

**Comment by madhurgupta10** [comment]
@nicolas-raoul Let us know if this PR is ready to be merged :)

**Review by nicolas-raoul — APPROVED** [review]

**Comment by nicolas-raoul** [comment]
so fast haha

**Pull request merged**